src/components/form/form.stories.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
I am unable to enter text into the input.

![image](https://user-images.githubusercontent.com/26925206/53956845-afe86c00-40d4-11e9-9373-731e9c57feca.png)